runtime/doc/syntax.txt

```diff
@@ -930,6 +930,27 @@ nasm_loose_syntax	unofficial parser allowed syntax not as Error
 nasm_ctx_outside_macro	contexts outside macro not as Error
 nasm_no_warn		potentially risky syntax not as ToDo
 
+ASTRO						*astro.txt* *astro-configuration*
+
```

Comment:
That might be better.

```
ASTRO			    *astro.vim* *ft-astro-syntax* *astro-configuration*

Configuration                                         

The following variables control certain syntax highlighting features.
You can add them to your .vimrc:
>
	let g:astro_typescript = "enable"
<
Enables TypeScript and TSX for ".astro" files. Default Value: "disable"
>
	let g:astro_stylus = "enable"
<
Enables Stylus for ".astro" files. Default Value: "disable"

NOTE: You need install an external plugin to support stylus in astro files.
```
Please note that I only corrected the style, not the content.

It might be better to give some description, introductions and the like.
For example, what this phrase means:
"NOTE: You need install an external plugin to support stylus in astro files."